src/domain/collaboration/callout/CalloutsPage.tsx

```diff
@@ -18,10 +18,11 @@ import useCallouts from './useCallouts';
 interface CalloutsPageProps {
   entityTypeName: EntityTypeName;
   rootUrl: string;
+  scrollToCallout?: boolean;
```

Comment:
Do you really need this parameter?

Reply:
yes, because when loading a Canvas, for example we do have the parameter calloutNameId available in the url.
We could check if we have a calloutNameId and not have an AspectNameId/CanvasNameId, but I think it's nicer to check at Routing level if we have to scroll or not

Reply:
But you are not performing any checks. At the routing level, you are saying `scrollToCallout=true`